site/_apps/cxjs-hackernews.txt

```diff
@@ -0,0 +1,27 @@
+---
+weight: 15
```

Comment:
Could you please bump this number up to the weight of the most recent contribution  + 1?